app/views/planning_applications/review/tasks/index.html.erb

```diff
@@ -14,27 +14,80 @@
 <%= render "shared/dates_and_assignment_header" %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-0 govuk-!-margin-top-5">
-      <%= t(".application_details") %>
-    </h2>
+  <div class="govuk-grid-column-full">
     <%= render(
-          AccordionComponent.new(planning_application: @planning_application)
+          SiteMapComponent.new(
+            planning_application: @planning_application
+          )
         ) %>
+  </div>
+</div>
 
-    <% if @planning_application.application_type.consultation? %>
-      <%= render "review_consultation" %>
-    <% end %>
+<div class="status-bar-container govuk-!-margin-bottom-0 govuk-!-margin-top-5" id="applicant-information-details">
```

Comment:
A while ago you suggested we turn these status panels into a view component — I think now is the right time to do so :)

(not in this PR but as part of this redesign anyway)

Reply:
I agree about making them in a view component 😄